internal/validators/validators.go

```diff
@@ -0,0 +1,22 @@
+package validators
+
+import (
+	"fmt"
+
+	"github.com/mattermost/awat/model"
+)
+
+type Validator interface {
+	Validate(archiveName string) error
+}
+
+func NewValidator(archiveType model.BackupType) (Validator, error) {
+	switch archiveType {
+	case model.MattermostWorkspaceBackupType:
```

Comment:
We always validate with `MattermostWorkspaceBackupType` for any archiveType. Then why do we need this `switch case`. We can simply return `NewMattermostValidator`. Any possibility for future usage?

Reply:
I left it there in case we add a simple Slack (or any other) validator before a translation.